kube-core/src/request.rs

```diff
@@ -268,6 +274,9 @@ impl Request {
         if pp.dry_run {
             qp.append_pair("dryRun", "All");
         }
+        if let Some(ref fm) = pp.field_manager {
+            qp.append_pair("fieldManager", fm);
+        }
```

Comment:
The docs for fieldManager (in link you posted) mention it is for all POST/PUT/PATCH so we should probably do this for:

- create_subresource
- patch_subresource

as well.

Reply:
Done! I copied the idea of `populate_qp` as used on PatchParams to reduce some duplication. Is that ok with you?

Reply:
Yeah, that is cleaner. Thank you!